app/controllers/revisions_controller.rb

```diff
@@ -11,6 +11,7 @@ def index
   end
 
   def show
+    @record_class = @record.class.name.downcase
```

Comment:
```suggestion
    @class_name = @record.class.name.downcase
```
For naming consistency with: https://github.com/dradis/dradis-ce/blob/develop/app/views/nodes/items_table/_table.html.erb#L1